source/get-started/cloud-install/gce.rst

```diff
@@ -229,12 +229,10 @@ Setup |CL| VM on GCP
 
         Figure 18: Set SSH key for remote login
 
-     .. warning::
+     .. note::
 
         The username is assigned from characters preceding ``@`` in the email
-        address, included in the SSH key. The dot symbol "." is not allowed,
-        because it is an invalid character while creating user accounts in
-        |CL|.
+        address, included in the SSH key.
```

Comment:
Hi @ttzeng. On GCP, has this changed to allow the '.' (dot) in the username?

Reply:
@mvincerx , the limitation of '.' (dot) in the username was imposed in Clear Linux, and it's fixed in the later releases. (ref: [Issue#829](https://github.com/clearlinux/distribution/issues/829) of CL distribution repo).

Reply:
Thanks for the clarification.